dxp/liferay_frontend_component.md

```diff
@@ -0,0 +1,28 @@
+# Demystifying `liferay-frontend:component` tag
+
+We can use this tag for removing JavaScript code from JSPs. This tag accepts a `module` property through which We can pass a relative path for the script to be loaded. Also, this tag registers automatically the component to [Liferay’s component global registry](https://github.com/liferay/liferay-frontend-guidelines/blob/bc6dae8514af04a6384a7bea9d4ddf266087ffc5/dxp/liferay_component.md#register), depending on the `componentId` property. When using this tag, `namespace`(or `portletNamespace`) and `spritemap` props will become available.
```

Comment:
1. We -> we

2. registers automatically the component to -> registers automatically the component with

Regarding 2, I think that's the correct form (see https://idioms.thefreedictionary.com/register+with), however I would like to invoke @wincent 's wildcard here :sweat_smile:

Reply:
Yes, "with" is better than "to" in this case, and you can additionally make the flow sound more naturally by changing the order of the words:

eg. "registers automatically the component with" → "automatically registers the component with"

Reply:
Thanks for the feedback <3